src/P2P/Node/RestAPI/Server.hs

```diff
@@ -95,6 +95,7 @@ peerGetHandler db nid limit next = do
         . SP.zip (SP.each [0..])
         . SP.each
         . toAscList (Proxy @HostAddressIdx)
+        . getEQ (SuccessiveFailures 0)
```

Comment:
Only gossip around nodes that didn't have any recent failure.

(Is this too strict? Should we allow nodes with, say, one successive failure?)

Reply:
Perhaps. @gregorycollins ?